### PR TITLE
contribution section is updated

### PR DESCRIPTION
**Pull request by palashg7563**
@abhishek71994 This pull request is regarding issue #34 how to start the frontend and backend

### CODE REVIEW

**Comment by abhishek71994** [comment]
In reference to #34?

**Comment by palashg7563** [comment]
Yes as in this pull request i have updated the contribution.md how to start the project

**Comment by abhishek71994** [comment]
Can you please change the readme.md as well?
Updating the 
```
* Prerequisites
* Installation
* Run locally
* Built With
```

File: contributing.md [hunk 1]
```diff
@@ -4,6 +4,34 @@ When contributing to this repository,please first discuss the change you wish to
...
+yarn
+
+yarn start
+```
+
+This will start your node server at 4000 port
+
 ## Pull Request Process
 
 1. Create a fork
```
Comment (aditya81070) [inline contributing.md]: Fix this ordered list

**Comment by abhishek71994** [comment]
@palashg7563 can you please remove the changes done to the files that are not related to this issue, please?

**Comment by palashg7563** [comment]
@abhishek71994

**Pull request closed without merging**